index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>Portfolio Sneak Peak</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <img class="status-bar" src="images/status-bar.png" alt="Portfolio Status Bar">
+      <nav>
+        <div class="logo">
+          <img src="images/app-bar.png" alt="Logo image">
+        </div>
+      </nav>
+    </header>
+    <section>
+      <h1>I’m Sylvester,
+          Glad to see you!
+      </h1>
+    </section>
+    <section>
+      <p class="intro">
+        I’m a software developer! I can help you build a product, 
+        feature or website. Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
+      <div class="social-icons">
+        <ul>
+          <li>
+            <a href="#">
+              <img src="images/twitter.png" alt="Connect on Twitter">
+           </a>
+          </li>
```

Comment:
- [ ] The `section` on lines `22 - 26` can be one section since the first one only has an `h1` tag in it.